### PR TITLE
Add dedicated Get Involved Page

### LINKED ISSUE #3: MVP Get Involved Page
Currently, the get involved section is where small on the main page.
We want to turn that into something bigger and also honour all our fabulous contributors.

## Tasks
- [x] Create Contributors Showcase Component (#38) 
- [x] Brainstorm ways for people to get involved
- [ ] Stretch goal: Automatically merge all repositories contributors via cron like Github Action
- [ ] Have a [list of good first issues](https://github.com/search?utf8=✓&q=user%3Aopenclimatefix+label%3A"good+first+issue"&type=Issues&ref=advsearch&l=&l=) on there

## Mockup
image

### PR DESCRIPTION
**Pull request by drwm-base**
I have added the Get Involved Page by creating a new page, migrating the get involved section for the main page, and linking the new page to the Get Involved button. The new page also contains the Contributor Showcase component, which I modified to fit the mockup better. Finally, I have added a link to good first open issues, and a space for anything OpenClimateFix wants to say to potential contributors.

Fixes #3.

### CODE REVIEW

**Comment by flowirtz** [comment]
Looks amazing, thanks so much @drwm-base!!
Sorry for being so slow to respond, I bricked my node in the meantime and only just now got it back to work. Will review now asap.

File: src/pages/get-involved.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          Jack&apos;s blog post about getting involved
+        </a>
+        .
+      </p>
+      <h2>Contribute to one of our repositories on Github</h2>
+      <p>
+        Check out our
+        {' '}
+        <a href="https://github.com/search?l=&p=1&q=user%3Aopenclimatefix+label%3A%22good+first+issue%22&ref=advsearch&type=Issues&utf8=%E2%9C%93&state=open">List of Good First Issues</a>
+        to get started!
```
Comment (flowirtz) [inline src/pages/get-involved.jsx:40]: need another space here after the link, i.e. `{' '}`

**Review by flowirtz — CHANGES_REQUESTED** [review]
One teeny tiny comment, apart from that good to go!
Really pleased with the result. Thanks a bunch, for working on it, David!

File: src/pages/get-involved.jsx (hunk 1 above)
Comment (drwm-base) [reply src/pages/get-involved.jsx:40]: just fixed

Comment (flowirtz) [reply src/pages/get-involved.jsx:40]: jeeez so quick! Amazing, thanks!!

**Review by flowirtz — APPROVED** [review]

File: src/pages/get-involved.jsx (hunk 1 above)
Comment (drwm-base) [reply src/pages/get-involved.jsx:40]: Yeah it just popped up in my email

**Pull request merged**